transactions/src/main/java/org/jboss/as/txn/subsystem/TransactionExtension.java

```diff
@@ -124,14 +124,10 @@ public void initialize(ExtensionContext context) {
      */
     @Override
     public void initializeParsers(ExtensionParsingContext context) {
-        context.setSubsystemXmlMapping(SUBSYSTEM_NAME, Namespace.TRANSACTIONS_1_0.getUriString(), TransactionSubsystem10Parser::new);
```

Comment:
@kabir I am not sure why we removed these but I think the removal causes the manualmode test to fail, I will add them back and see if it passes again